webapp/javascript/components/FlameGraph/FlameGraphComponent/Flamegraph_render.ts

```diff
@@ -79,6 +82,7 @@ export default function RenderCanvas(props: CanvasRendererConfig) {
   const { units } = props;
   const { rangeMin, rangeMax } = props;
   const { tickToX } = props;
+  const { palette } = props;
```

Comment:
just curious, could all these be combined into one? Any particular reason they are not? 

i.e. 
```
const {
    canvas,
    numTicks,
    sampleRate ,
    pxPerTick,
    fitMode,
    units,
    rangeMin ,
    rangeMax,
    tickToX ,
    palette
} = props;
```